pkgs/build-support/fetchgithub/default.nix

```diff
@@ -11,11 +14,16 @@ lib.makeOverridable (
 , ... # For hash agility
 }@args:
 
+assert (lib.assertMsg (lib.xor (tag == null) (rev == null)) "fetchFromGitHub requires either `rev` xor `tag`.");
```

Comment:
Nice and simple because there's no HEAD business. Same copyediting as before with the message.

Reply:
Once the HEAD issue is resolved, we can simply use this logic too.